.gitlab-ci.yml

```diff
@@ -0,0 +1,16 @@
+before_script:
+  - call berks
+  - call berks vendor
+  - move berks-cookbooks cookbooks
+  - echo done with berks
+  
+dd-agent:
+  script:
+    - chef-client -j c:\cfg\attributes.json -zo dd-agent-builder::build
```

Comment:
Can we add the `-l info` flag? It will print the log in real time instead of at the end of the build.